lib/providers/google.js

```diff
@@ -10,13 +10,16 @@ exports = module.exports = function (options) {
         scope: ['profile', 'email'],
         profile: function (credentials, params, get, callback) {
 
-            get('https://www.googleapis.com/plus/v1/people/me', null, (profile) => {
+            get('https://www.googleapis.com/oauth2/v3/userinfo', null, (profile) => {
 
                 credentials.profile = {
                     id: profile.id,
-                    displayName: profile.displayName,
-                    name: profile.name,
-                    emails: profile.emails,
+                    displayName: profile.name,
+                    name: {
+                        given_name: profile.given_name,
+                        family_name: profile.family_name
```

Comment:
I don't have a problem with following that convention if that's what @ldesplat wants, but that will make this object different to Google Plus's api, which uses underscores. 

This is important because anyone upgrading from the previous version of Bell to this one will have to change their code to be snake case as well. Choosing code convention over API compatibility seems like the wrong call IMO.